src/EnergyPlus/IntegratedHeatPump.cc

```diff
@@ -1967,119 +1967,119 @@ void GetIHPInput(EnergyPlusData &state)
         state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).MaxHeatAirVolFlow = 1e10;
         state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).MaxCoolAirMassFlow = 1e10;
         state.dataIntegratedHP->IntegratedHeatPumps(DXCoilNum).MaxCoolAirVolFlow = 1e10;
-    }
 
-    if (ErrorsFound) {
-        ShowFatalError(state,
-                       std::string{RoutineName} + "Errors found in getting " + CurrentModuleObject +
-                           " input.  Preceding condition(s) causes termination.");
-    } else {
-        // set up output variables, not reported in the individual coil models
-        SetupOutputVariable(state,
```

Comment:
I moved this chunk into the `for` loop. The rest is indentation formatting.

Reply:
So E+ will fatal out on the first IHP with an input error? i.e., ErrorsFound will be true for that IHP and the rest will not be read in. We typically loop through all instances of an equipment type to allow the warnings to print to the error file so the user can fix all at once instead of one at a time. It would be better to move this back and for loop inside the else to set up report vars for all IHPs.

Reply:
Thanks, @rraustad. I moved the fatal error call outside of the get input and SetupOutputVars now happens all at once after that.